src/tests/operate.test.js

```diff
@@ -0,0 +1,55 @@
+// eslint-disable-next-line
```

Comment:
- [ ] Please don't disable eslint checks. We use Eslint to keep to a certain standard of the Javascript we write and disabling a check deviates from that.